module_utils/helpers.py

```diff
@@ -334,10 +400,27 @@ def filter_none_values(d):
     }
 
 
+def get_cluster_instances(control_console):
+    instances, _ = control_console.eval_res_err(GET_INSTANCES_FUNC_BODY)
+
+    return instances
+
+
+def get_cluster_replicasets(control_console):
+    cluster_replicasets, _ = control_console.eval_res_err(GET_REPLICASETS_FUNC_BODY)
+
+    if not cluster_replicasets:
+        cluster_replicasets = dict()
+
+    return cluster_replicasets
+
+
 class Helpers:
     DEFAULT_RUN_DIR = DEFAULT_RUN_DIR
     DYNAMIC_BOX_CFG_PARAMS = DYNAMIC_BOX_CFG_PARAMS
     MEMORY_SIZE_BOX_CFG_PARAMS = MEMORY_SIZE_BOX_CFG_PARAMS
+    FORMAT_SERVER_FUNC = FORMAT_SERVER_FUNC
+    FORMAT_REPLICASET_FUNC = FORMAT_REPLICASET_FUNC
```

Comment:
Is `GET_INSTANCES_FUNC_BODY` and `GET_REPLICASETS_FUNC_BODY` only for internal usage? If no, then let's do:
```suggestion
    FORMAT_REPLICASET_FUNC = FORMAT_REPLICASET_FUNC
    GET_INSTANCES_FUNC_BODY = GET_INSTANCES_FUNC_BODY
    GET_REPLICASETS_FUNC_BODY = GET_REPLICASETS_FUNC_BODY
```

Reply:
This constants aren't used by other modules, so there is no reason to export them.